### ISSUE TITLE
 multi-var statement gets mangled [regression in 0.22.1]

### ISSUE THREAD

**Issue by lukeapage** [rollup/rollup#390]
Unfortunately the code is closed source so I'll try and paint a picture of what is going on..

I'm using format iife and the `rollup-plugin-babel` all latest versions.

I have code like..

```
import mockFetch from '../../mocks/fetch';

var a, fetch;

describe("x", () => {
	beforeEach(() => {
		fetch = mockFetch();
		installClock();
	});
```

fetch is a global and is used as a global in mockFetch, so I guess rollup needs to rename the `var fetch`

but it becomes

```
	var a;
	fetch$1;
	describe("x", function () {
		beforeEach(function () {
			fetch$1 = mockFetch();
			installClock();
		});
```
which means that `fetch$1` isn't defined and the code falls over.

This was working a couple of weeks ago so I'll try and figure out what version causes the issue.

**Comment by lukeapage** [comment]
ok thats simple, this occurs in 0.22.1 but *not* 0.22.0.

In 0.22.0 I get the following output

```
var a;
var fetch$1;
```

**Comment by kobezzza** [comment]
:+1: 

**index.js**

```js
import './a';
import './b';
```

**a.js**

```js
const a = 1, b = 2;

console.log(a, b);

```

**b.js**

```js
const a = 2, b = 3;

console.log(a, b);
```

**Command:**

```bash
rollup rollup/index.js --export umd --id foo --name foo
```
**Output:**

```js
const a = 1;
const b = 2;
console.log(a, b);

a$1 = 2; // Error
b$1 = 3; // Error
console.log(a$1, b$1);
```

**Comment by lukeapage** [comment]
thanks @kobezzza for the repro-case

**Comment by Rich-Harris** [comment]
Thanks for flagging this up. Looks like the latest version of magic-string is to blame – looking into it

**Comment by Rich-Harris** [comment]
[Fixed in 0.22.2](http://rollupjs.org/#%7B%22options%22%3A%7B%22format%22%3A%22umd%22%2C%22moduleName%22%3A%22myBundle%22%2C%22globals%22%3A%7B%7D%2C%22moduleId%22%3A%22%22%7D%2C%22modules%22%3A%5B%7B%22name%22%3A%22main.js%22%2C%22code%22%3A%22import%20'.%2Fa.js'%3B%5Cnimport%20'.%2Fb.js'%3B%22%7D%2C%7B%22name%22%3A%22a.js%22%2C%22code%22%3A%22const%20a%20%3D%201%2C%20b%20%3D%202%3B%5Cn%5Cnconsole.log(a%2C%20b)%3B%22%7D%2C%7B%22name%22%3A%22b.js%22%2C%22code%22%3A%22const%20a%20%3D%202%2C%20b%20%3D%203%3B%5Cn%5Cnconsole.log(a%2C%20b)%3B%22%7D%5D%7D)
